{"status": "broken", "description": "Activating the pick action test and checking whether it was successful.\n        ", "labels": [{"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "complex_test"}, {"name": "subSuite", "value": "TestComplexScenario"}, {"name": "host", "value": "Hyper-Dock"}, {"name": "thread", "value": "22669-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython2"}, {"name": "package", "value": "tests.complex_test"}], "stop": 1608373852327, "uuid": "fbe07db4-95b0-4aad-b363-750e9dff6f20", "historyId": "6df5cbf7c00ce5f14b4c42d69bcd2de9", "name": "test_verification_of_pick_action", "start": 1608373851842, "fullName": "tests.complex_test.TestComplexScenario#test_verification_of_pick_action", "statusDetails": {"message": "RuntimeError: Group 'arm' was not found.", "trace": "self = <tests.complex_test.TestComplexScenario instance at 0x7f302d374560>\n\n    def test_verification_of_pick_action(self):\n        \"\"\"Activating the pick action test and checking whether it was successful.\n        \"\"\"\n        data_logger('logger/logs/pick_action_start')\n        # Pick from table and place on table\n        result = MoveItPickAndPlace(pick_x = -0.03, pick_y = 0.96 , pick_z = 0.57,\n>                                   place_x = 0.5, place_y = 1.0 , place_z = 0.6)\n\ntests/complex_test.py:173: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \ntests/action_client/pick_place_client.py:27: in __init__\n    arm = moveit_commander.MoveGroupCommander(\"arm\")\n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nself = <moveit_commander.move_group.MoveGroupCommander object at 0x7f302dfe0090>\nname = 'arm', robot_description = 'robot_description', ns = ''\n\n    def __init__(self, name, robot_description=\"robot_description\", ns=\"\"):\n        \"\"\" Specify the group name for which to construct this commander instance. Throws an exception if there is an initialization error. \"\"\"\n>       self._g = _moveit_move_group_interface.MoveGroupInterface(name, robot_description, ns)\nE       RuntimeError: Group 'arm' was not found.\n\n/opt/ros/kinetic/lib/python2.7/dist-packages/moveit_commander/move_group.py:52: RuntimeError"}, "attachments": [{"source": "325f6fec-c71e-4f89-b2c7-3d8cdb64deef-attachment.txt", "type": "text/plain", "name": "log"}, {"source": "274b48ec-4fc5-4270-87fb-595e4fcb52d3-attachment.txt", "type": "text/plain", "name": "stdout"}, {"source": "e67d1255-27fe-40bb-b8b2-1df7be66016b-attachment.txt", "type": "text/plain", "name": "stderr"}]}